src/styles/blocks/card.scss

```
@@ -0,0 +1,145 @@
+.card {
+
+  box-sizing: border-box;
```

```suggestion
.card {
  box-sizing: border-box;
```
Better add box-sizing for all elements(*) so you don't need to change it later